### PR TITLE
Don't use commit-pin-updates, replace with trigger-auto-pin-update-process

### PR DESCRIPTION
**Pull request by Brian-McM**
commit-pin-updates is deprecated, we are no longer doing the pin updates and pushing them directly to master, we're updating them via a PR that will run the ci and get merged automatically

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by bcreane — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
